code/modules/mob/living/carbon/alien/damage_procs.dm

```diff
@@ -1,21 +1,21 @@
 
-
-/mob/living/carbon/alien/getToxLoss()
-	return 0
-
-/mob/living/carbon/alien/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE) //alien immune to tox damage
```

Comment:
You have removed the toxin damage immunity procs without replacing them. You need to **test** that absolutely nothing bypasses `apply_damage()` to call `adjustToxLoss`. Failing to do this will mean that you would be allowing certain sources of toxin damage to apply to the Xeno when she is meant to be entirely immune.